modules/oadp-configuring-dpa-multiple-bsl.adoc

```diff
@@ -16,7 +28,7 @@ You can configure the DPA with more than one BSL and specify the credentials pro
 
 .Procedure
 
-. Configure the DPA with more than one BSL. See the following example.
+. Configure the `DataProtectionApplication` with more than one `BackupStorageLocation`. See the following example:
```

Comment:
```suggestion
. Configure the `DataProtectionApplication` CR with more than one `BackupStorageLocation` CR. See the following example:
```

- Object name is needed: [guideline](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references)

> An object reference is when you are referring to the actual instance of an API object, where the object name is important.